pypowsybl/network.py

```diff
@@ -201,6 +207,13 @@ def name(self) -> str:
         """
         return self._name
 
+    @property
+    def objects(self) -> str:
```

Comment:
I don't think adding this property is useful, it's almost a duplicate of the string representation

Reply:
@sylvlecl 
Thank you for the review, I do understand your point but the motive here is to inform the user about the network elements.
The old string representation had just the meta info.
The issue of not updated elements is right , that has to be dynamically done
How do you suggest that?